docs/local-setup/running-a-node.md

```diff
@@ -125,13 +120,13 @@ If you are running a validator in production you may find it more efficient to j
 cargo build -p neard --release
 ```
 
-NB. Please ensure you include the `--release` flag. Omitting this will lead to an unoptimized binary being produced that is too slow for a validator to function effectively.
+**Note**: Please ensure you include the `--release` flag. Omitting this will lead to an unoptimized binary being produced that is too slow for a validator to function effectively.
 
 Finally:
 On MacOS or Linux
 
 ```bash
-nearup betanet --nodocker --binary-path path/to/nearcore/target/release
+nearup betanet --nodocker --binary-path <path_to_nearcore_target_release>
```

Comment:
this is outdated. The new version of nearup doesn't have an option to run docker inside. You can run the entire nearup from docker. https://github.com/near/nearup/tree/nearup_v2

Reply:
Same as above... existing instructions on `nearup` were not checked for accuracy. I will carve out some time to run through them and update.